ynput/ayon-core
# Maya: Yeti - Implement writing and loading user variables with a yeti cache

## Changelog Description

Implement writing and loading user variables with a yeti cache

## Additional info

Support writing and loading of user variables in a yeti node with the `yeticache` product type.

## Testing notes:

1. Loading legacy yeti products should work
2. Publish a new yeti product without user variables should work + loading it + updating it
3. Publish a new yeti product with user variables should work + loading it + updating it
